igvc_navigation/launch/localization.launch

```diff
@@ -59,25 +58,25 @@
                X velocity, Y velocity, Z velocity,
                roll velocity, pitch velocity, yaw velocity,
                X acceleration, Y acceleration, Z acceleration -->
-      <rosparam param="odom0_config">[false,  false, false,
-                               false, false, false,
-                               false, false, false,
-                               false, false, false,
-                               false, false, false]</rosparam>
+      <rosparam param="odom0_config">[false, false, false,
+                                      false, false, false,
+                                      true, true, false,
+                                      false, false, false,
```

Comment:
Is yaw rate intentionally off?

Reply:
Yes. wheel odometry yaw is, I suspect, less accurate than the yaw supplied by the gryoscope.

Reply:
Yes it should be less accurate. It will also help prevent drift. how much did you play with the covariance matrix?